baselines/pFedHN/pyproject.toml

```diff
@@ -0,0 +1,141 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "pFedHN" # <----- Ensure it matches the name of your baseline directory containing all the source code
+version = "1.0.0"
+description = "Personalized Federated Learning using Hypernetworks"
+license = "Apache-2.0"
+authors = ["The Flower Authors <hello@flower.dev>"]
```

Comment:
Please add more entries to the `authors` (using the same format) with your name+emails

Reply:
Take care